heron/instance/src/cpp/gateway/stmgr-client.cpp

```diff
@@ -97,32 +98,35 @@ void StMgrClient::HandleClose(NetworkErrorCode code) {
   AddTimer([this]() { this->OnReconnectTimer(); }, reconnect_interval_ * 1000 * 1000);
 }
 
-void StMgrClient::HandleRegisterResponse(void*, proto::stmgr::RegisterInstanceResponse* response,
-                                         NetworkErrorCode status) {
+void StMgrClient::HandleRegisterResponse(
+                                        void*,
```

Comment:
indent

Reply:
Here and in other "indent" comments, if i align the indent, then the code in the line will hit the max line limit (100 chars) and, as a result, a compilation fails on a style-check step...

Reply:
I see. In that case, it might be better to be 4-space indent:

void StMgrClient::HandleRegisterResponse(
    void*,
    proto::stmgr::RegisterInstanceResponse* response,
    NetworkErrorCode status) {
  // implementation
}

Reply:
fixed